test/ArrayMathTest.java

```diff
@@ -25,10 +25,21 @@ public void testDotProductTinyVectors() {
 		y = new double[] { };
 		assertEquals( 0.0, ArrayMath.dotProduct(x, y), TOL);
 	}
-	
-	//TODO Add at least one test for the "typical" case: vectors larger than 1.
-	// Please don't copy my tests. You don't need random numbers (not a good idea
-	// because test results may not be reproducable).
+
+	@Test
+	public void testTypicalVectors() {
+		double[] x = {1, 2, 3};
+		double[] y = {1, 5 ,7};
+		double expected = 32.0;
+		assertEquals(expected, ArrayMath.dotProduct(x, y), TOL);
+		assertEquals(expected, ArrayMath.dotProduct(y, x), TOL);
+
```

Comment:
Put in another method

Reply:
d4b6695f64f74165aaa10c2142dc91084347b4dc